.env.example

```diff
@@ -25,13 +25,13 @@ HASURA_GRAPHQL_UNAUTHORIZED_ROLE=anonymous
 HASURA_GRAPHQL_ACTION_BASE_URL=http://hapi:9090
 
 # WEBAPP
-REACT_APP_GRAPHQL_HTTP_URL=http://localhost:8080/v1/graphql
-REACT_APP_API_URL=https://jungle.eosio.cr
-REACT_APP_GRAPHQL_WS_URL=ws://localhost:8080/v1/graphql
-REACT_APP_EOS_API_URL=https://jungle.eosio.cr
-REACT_APP_EOS_API_HOST=jungle.eosio.cr
-REACT_APP_EOS_API_PORT=443
-REACT_APP_EOS_API_PROTOCOL=https
-REACT_APP_EOS_CHAIN_ID=2a02a0053e5a8cf73a56ba0fda11e4d92e0238a4a2aa74fccf46d5a910746840
-REACT_APP_BLOCK_EXPLORER=https://jungle3.bloks.io
-REACT_APP_NETWORK_MONITOR_URL=https://jungle.eosio.online/
\ No newline at end of file
+WEBAPP_GRAPHQL_HTTP_URL=http://localhost:8080/v1/graphql
+WEBAPP_API_URL=https://jungle.eosio.cr
```

Comment:
@xavier506 should I remove it, right? is not been used except by `webapp/Dockerfile`